.circleci/config.yml

```diff
@@ -50,14 +50,6 @@ jobs:
             - "ee:b4:00:e6:23:5b:55:bb:fd:07:bc:73:9e:f7:89:9c" # shared.aspire@talis.com 'talis-cdk-constructs Deploy Key'
       - checkout
       - node/install-packages
-      - run:
-          name: Define environment variable with lastest commit's message
-          command: |
-            echo 'export COMMIT_MESSAGE=$(git log -1 --pretty=format:"%s")' >> $BASH_ENV
-            source $BASH_ENV
-      - run:
-          name: Lint commit message
-          command: echo "$COMMIT_MESSAGE" | npx commitlint
```

Comment:
Still causing unnecessary build failures: https://app.circleci.com/pipelines/github/talis/talis-cdk-constructs/307/workflows/708f533e-5ba7-456e-b68c-9a5c19731179/jobs/694

This was meant to have been removed here
 - https://github.com/talis/platform/issues/5173